src/arch/ArchHooks/ArchHooks_Win32Static.cpp

```diff
@@ -22,30 +22,35 @@ static bool g_bTimerInitialized;
  * QueryPerformanceCounter and QueryPerformanceFrequency expect
  * a LARGE_INTEGER, which is a union. These functions store data
  * in the QuadPart of the LARGE_INTEGER, which is a 64-bit integer. */
-LARGE_INTEGER g_liFrequency, g_liStartTime, g_liCurrentTime;
+namespace {
+    LARGE_INTEGER g_liFrequency;
+    LARGE_INTEGER g_liCurrentTime;
+} // namespace
```

Comment:
Minor nit but two spaces after between code and comments

Reply:
```
namespace {
    LARGE_INTEGER g_liFrequency;  //  namespace
    LARGE_INTEGER g_liCurrentTime;  //  namespace
}
```

?

Reply:
```cpp
namespace {
    LARGE_INTEGER g_liFrequency;
    LARGE_INTEGER g_liCurrentTime;
}  // namespace
```

Reply:
Two spaces before `//`

Reply:
It is done 🙏

Reply:
https://google.github.io/styleguide/cppguide.html#Internal_Linkage

Reply:
> https://google.github.io/styleguide/cppguide.html#Internal_Linkage


Thank you for this!